### ISSUE TITLE
Better Contributing.md file

### ISSUE THREAD

**Issue by YashM246** [jcubic/Clarity#17]

**Comment by YashM246** [comment]
Hey I can add some more content to the contributing.md file. If possible could you assign this to me under the Hacktober label?

**Comment by jcubic** [comment]
What exactly do you want to add to the file? I can assign the issue to you don't need any label, as far as I know. You only need a merged PR and topic on the repo. If PR is not yet merged it can have the hacktoberfest-accepted label.

**Comment by YashM246** [comment]
@jcubic Your contributing.md file only shows how to contribute icons. I think it should also contain steps on how to exactly contribute i.e. forking, cloning and making a merge/pull request. Also I think adding a Hacktober label is better, else it shows as Not-Participating even if you merge with your main repo.
Let me know if you are going to assign this.

**Comment by jcubic** [comment]
I think this is kind of stupid to add information about how to clone and use git in the contribution file. When you go to the repo you should know this already. If you don't know that you need to create a fork I think that contribution to Open Source on GitHub is not for you.

I don't want information about forking or how to use git in the Contribution file. The contribution file is not a tutorial on how to use GitHub.

**Comment by YashM246** [comment]
Considering you posted your repository link on the Hacktober fest Discord, I assumed you wouldn't be partial towards people who are beginners at contributing in Open Source Projects. 
Hacktober fest is not only a celebration of Open Source Projects and the Community but also an event to encourage new people to get into Open Source Contributing.

Hence I thought it would be beneficial to add these details to the contributing.md file.

> I think this is kind of stupid to add information about how to clone and use git in the contribution file. When you go to the repo you should know this already.

If this your final view, you can close this issue. 
Good Day.

**Comment by Saket2** [comment]
I disagree with you @YashM246 on 'being partial' part. It's his repository and he can decide what to put/not put. How to fork/clone etc is easily available to Google too - appears in the results on the 1st search. Not deciding to put your desired information doesn't make it less encouraging to new people. Probably it only becomes discouraging to people like you who just want to add up one PR to make it count towards 4 PRs.

**Comment by YashM246** [comment]
@Saket2 

>It's his repository and he can decide what to put/not put.

I know that's why I mentioned that he can close this issue. 

>  Not deciding to put your desired information doesn't make it less encouraging to new people.

Lets not start a debate/argument here. I don't even understand why you felt the need to comment this?  I know you disagree on my opinion and that's completely fine! Lets just continue with our open source journey without starting an argument with random people on the internet.

>Hence I thought it would be beneficial to add these details to the contributing.md file.

Here I just stated my view/opinion. There was no need for further appeals/opinions. The owner of this repository, as you said can decide what to do further.

> Probably it only becomes discouraging to people like you who just want to add up one PR to make it count towards 4 PRs.

Again I don't even understand why someone would want to start an argument by saying stuff like this. Anyone can go to my profile and see the number of commits I have made since the start of the month, which for your information is way more than the required 4 PRs. If I only wanted to add up the count towards 4 PRs, I would have stopped days ago! 

Considering @Saket2 only started contributing since October 8, wouldn't you like to save yourself some time by not inciting arguments with people over the internet and helping the community?

Good Day.
